foundation/foundation-mda/src/main/resources/templates/integration-test/it.pipeline.steps.vm

```diff
@@ -1,127 +1,42 @@
-import org.junit.Assert;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-import java.io.File;
-import java.io.FileInputStream;
-import java.io.IOException;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Objects;
-import java.util.concurrent.TimeUnit;
 
 public class PipelineSteps {
 
     private static final Logger logger = LoggerFactory.getLogger(PipelineSteps.class);
-    List<File> listOfFiles = new ArrayList<>();
-    Boolean jobRequestFailed;
-    List<Pod> podList;
-    List<HasMetadata> hasMetadata;
 
     @Before("@pipeline")
     public void setup() {
-        this.jobRequestFailed = false;
-        this.podList = new ArrayList<>();
-        this.hasMetadata = new ArrayList<>();
     }
 
     @After("@pipeline")
     public void cleanup() {
     }
 
-    @When("the job is triggered")
-    public void theJobIsTriggered() throws IOException, InterruptedException {
-        getFiles();
-    }
-
-    @Then("the pipeline is executed")
-    public void thePipelineIsExecuted() {
-        Assert.assertTrue(String.format("Pod list size should be greater than  %d", podList.size()), podList.size() > 0);
-    }
-
-    private void getFiles() throws IOException, InterruptedException {
-
-        File dir = new File("./pipelines");
-        addFilesForFolder(dir);
-
-        logger.info(String.format("Got %d application.yaml file(s) to process", listOfFiles.size()));
-
-        for (File file : listOfFiles) {
-
-          try (KubernetesClient client = new KubernetesClientBuilder().build()) {
-
-              //Initializing Informer
-              this.initializeInformer(client);
-
-              List<HasMetadata> metadata = client
-                  .load(new FileInputStream(file))
-                  .get();
-
-              this.hasMetadata.addAll(metadata);
-              logger.info(String.format("has meta data size is %d", hasMetadata.size()));
-
-              //Creating/Replacing resource here
-              metadata.forEach(o -> logger.info("Creating resource " + o.getMetadata().getName()));
-              client.resourceList(metadata).inNamespace("default").createOrReplace();
-              TimeUnit.MINUTES.sleep(2);
-
-          }
-        }
-    }
-
-    private void initializeInformer(KubernetesClient client) {
-
-        client.pods().inNamespace("default").inform(new ResourceEventHandler<>() {
-
-            @Override
-            public void onAdd(Pod pod) {
-                addToPodList(pod);
-            }
-
-            @Override
-            public void onUpdate(Pod oldPod, Pod newPod) {
-                addToPodList(oldPod);
-            }
-
-            @Override
-            public void onDelete(Pod pod, boolean deletedFinalStateUnknown) {
-                addToPodList(pod);
-            }
-
-        }, 30 * 1000L);
-
-        logger.info("Informer initialized.");
+    @Given("a precondition")
+    public void a_precondition() {
```

Comment:
I: I don't think this will be worth doing right now, but when we properly revisit the generated ITs, we should also consider including an execution of `-Pintegration-test` in the `archetypte-test` script so that we don't let it get to such a rough spot again.